examples/chip-tool/commands/common/CHIPCommand.cpp

```diff
@@ -83,6 +86,41 @@ CHIP_ERROR GetAttestationTrustStore(const char * paaTrustStorePath, const chip::
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR GetRevocationSet(const char * revocationSetPath, const chip::Credentials::RevocationSet ** revocationSet)
+{
+    bool failed = false;
+
+    if (revocationSetPath == nullptr)
+    {
+        revocationSetPath = getenv(kRevocationSetPathVariable);
+    }
+
+    static chip::Credentials::JSONFileRevocationSet fileRevocationSet{ revocationSetPath };
+
+    if (revocationSetPath == nullptr)
+    {
+        ChipLogError(chipTool, "No Revocation Sets provided.");
+        failed = true;
+    }
+    else if (fileRevocationSet.revocationSetCount() == 0)
+    {
+        ChipLogError(chipTool, "No Revocation Sets found in path: %s", revocationSetPath);
+        failed = true;
+    }
+
+    if (failed)
+    {
+        ChipLogError(chipTool,
```

Comment:
Won't this log "Please specify...." as an error on every single chip-tool invocation, even if the chip-tool user does not care about revocation sets?  That doesn't seem good.

Reply:
Agreed. Removing this statement.